partitioned-table.md

```diff
@@ -161,6 +161,192 @@ Range 分区在下列条件之一或者多个都满足时，尤其有效：
 * 使用包含时间或者日期的列，或者是其它按序生成的数据。
 * 频繁查询分区使用的列。例如执行这样的查询 `EXPLAIN SELECT COUNT(*) FROM employees WHERE separated BETWEEN '2000-01-01' AND '2000-12-31' GROUP BY store_id;` 时，TiDB 可以迅速确定，只需要扫描 `p2` 分区的数据，因为其它的分区不满足 `where` 条件。
 
+### List 分区
+
+> **警告：**
+>
+> 该功能从 5.0 GA 引入，目前为实验特性，不建议在生产环境中使用。
+
+在创建 List 分区表之前，需要先将 session 变量 `tidb_enable_list_partition` 的值设置为 `ON`。
+
+{{< copyable "sql" >}}
+
+```sql
+set @@session.tidb_enable_list_partition = ON
+```
+
+List 分区和 Range 分区有很多相似的地方。不同之处主要在于 List 分区中，对于表的每个分区中包含的所有行，按分区表达式计算的值属于给定的数据集合。每个分区定义的数据集合有任意个值，但不能有重复的值，可通过 `PARTITION ... VALUES IN (...)` 子句对值进行定义。
+
+假设你要创建一张人事记录表，示例如下：
+
+{{< copyable "sql" >}}
+
+```sql
+CREATE TABLE employees (
+    id INT NOT NULL,
+    hired DATE NOT NULL DEFAULT '1970-01-01',
+    store_id INT
+);
+```
+
+假如一共有 20 个商店分布在 4 个地区，如下表所示：
+
+```
+| Region  | Store ID Numbers     |
+| ------- | -------------------- |
+| North   | 1, 2, 3, 4, 5        |
+| East    | 6, 7, 8, 9, 10       |
+| West    | 11, 12, 13, 14, 15   |
+| Central | 16, 17, 18, 19, 20   |
+```
+
+如果想把同一个地区商店员工的人事数据都存储在同一个分区中，你可以根据 `store_id` 来创建 List 分区：
+
+{{< copyable "sql" >}}
+
+```sql
+CREATE TABLE employees (
+    id INT NOT NULL,
+    hired DATE NOT NULL DEFAULT '1970-01-01',
```

Comment:
Suggest updating the default hiring date to a newer date.
```suggestion
    hired DATE NOT NULL DEFAULT '2015-01-01',
```

Reply:
ditto.

Reply:
lgtm